src/constants/url.ts

```diff
@@ -0,0 +1,3 @@
+export const TINY_API_URL = 'https://tiny-site-backend.onrender.com/v1';
```

Comment:
What is the use case for this setup I have done if everyone does there own setup?
For frontend, you have the logs 
for backend use this : https://api-tinysite.onrender.com

Reply:
I will create a new ticket for the set the environment for different use cases like dev, staging and production

Reply:
ok